src/main/scala/me/rexim/morganey/module/Module.scala

```diff
@@ -8,16 +8,26 @@ import scala.util._
 
 import java.net.{URL, URLClassLoader}
 
-class Module(modulePath: ModulePath, classLoader: ClassLoader = Module.getClass.getClassLoader) {
+class Module(
+  modulePath: ModulePath,
+  preludeModule: Option[Module] = None,
```

Comment:
I'd like to start a discussion whether it'll be more useful to change the parameter order to `modulePath, classLoader, preludeModule`. It seems to me that it'll help to use the `preludeModule = None` default more often.